pom.xml

```
@@ -166,7 +166,7 @@
 			<plugin>
 				<groupId>org.eclipse.xtend</groupId>
 				<artifactId>xtend-maven-plugin</artifactId>
-				<version>2.27.0</version>
+				<version>2.32.0</version>
```

```suggestion
				<version>2.27.0</version>
```
Xtend requires Xtend libs, which aren't updated yet (as they depend on a new Eclipse platform).

---

ok, i have upgrade,  because 2.27 version not working with default function in interface. Now I have IMPROVE for this default function notice